flagsmith/manifest.json

```
@@ -12,17 +12,18 @@
     "title": "Flagsmith",
     "media": [],
     "classifier_tags": [
-      "Category::Configuration & Deployment",
-      "Category::Notification",
-      "Category::Testing",
       "Supported OS::Linux",
+      "Supported OS::macOS",
       "Supported OS::Windows",
-      "Supported OS::macOS"
+      "Category::Issue Tracking",
+      "Category::Developer Tools",
+      "Offering::UI Extension",
+      "Offering Type::Integration"
```

```suggestion
      "Offering::Integration"
```
Apologies for the confusion here, but this is the correct format